src/annotator/test/guest-test.js

```diff
@@ -385,34 +376,16 @@ describe('Guest', () => {
         };
       }
 
-      context('in an HTML document', () => {
-        it('calls the callback with HTML URL and metadata', done => {
-          fakeHTMLIntegration.uri.resolves('https://example.com/');
-          const metadata = { title: 'Example site' };
-          fakeHTMLIntegration.getMetadata.resolves(metadata);
-
-          guest = createGuest();
-
-          emitGuestEvent(
-            'getDocumentInfo',
-            createCallback('https://example.com/', metadata, done)
-          );
-        });
-      });
-
-      context('in a PDF document', () => {
-        it('calls the callback with PDF URL and metadata', done => {
-          fakeCreateIntegration.returns(fakePDFIntegration);
-          guest = createGuest();
-          const metadata = { title: 'hi' };
+      it('calls the callback with document URL and metadata', done => {
```

Comment:
What callback?

Reply:
This test is part of a larger block of tests for the "getDocumentInfo" RPC call from the sidebar.